src/Resolver/InstanceCreator.php

```diff
@@ -24,23 +28,38 @@ public function __construct(
      */
     public static function create(string $className): ?object
     {
-        return (new self())->createByClassName($className);
+        return (new self())->get($className);
+    }
+
+    public function has(string $id): bool
+    {
+        return is_object($this->get($id));
     }
 
     /**
      * @param class-string $className
+     *
+     * @deprecated Use method 'get(string $id)' instead
```

Comment:
I am deprecating this method, another alternative would be to make it an alias (basically remove the `@deprecate` tag from the PHPDoc).
I would go for removing this method in the future, but I would like to know your opinion @Chemaclass @Tito-Kati